modules/nativoBidAdapter.js

```diff
@@ -615,6 +703,61 @@ function getLargestSize(sizes, method = area) {
   })
 }
 
+/**
+ * Fire a Nativo Error AJAX request to log errors
+ * @param {String} errorMessage - The error message to send
+ */
+function fireNativoError(errorMessage) {
```

Comment:
@ChrisHuie What's the stance on an additional endpoint call for error tracking from a module, should this be behind a functionality flag for publishers to determine it's usage? Is there another suggested pattern for this?

Reply:
Hi @ncolletti and @ChrisHuie, I am also curious to the answer above. After speaking with @dgirardi below it seems like we may be able to use this logic as we can assume each auction's `bidRequest.userIdAsEids` will be the same. However, the idea of being able to log errors still sounds like it could be helpful in the future if this aligns with Prebid's adapter rules.

Reply:
I have removed it for now.